specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,736 @@
+          "type": "string"
+        },
+        "name": {
+          "description": "Custom container name.",
+          "type": "string"
+        },
+        "command": {
+          "description": "Container start command.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "args": {
+          "description": "Container start command arguments.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "env": {
+          "description": "Container environment variables.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/EnvironmentVar"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "resources": {
+          "$ref": "#/definitions/SessionContainerResources",
+          "description": "Container resource requirements."
+        }
+      }
+    },
+    "SessionIngress": {
+      "description": "Session Pool Ingress configuration.",
+      "type": "object",
+      "properties": {
+        "targetPort": {
+          "format": "int32",
+          "description": "Target Port in containers for traffic from ingress",
+          "type": "integer"
+        }
+      }
+    },
+    "SessionAuthConfiguration": {
+      "description": "Auth configuration of the Sessions in this Session Pool.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Auth status for the Sessions.",
+          "type": "string",
+          "enum": [
+            "authEnabled",
+            "authDisabled"
+          ],
+          "x-ms-enum": {
+            "name": "SessionAuthStatus",
+            "modelAsString": true
+          }
+        },
+        "jwtToken": {
+          "description": "JWT token.",
+          "type": "string"
+        }
+      }
+    },
+    "CustomContainerTemplate": {
+      "description": "Custom container configuration.",
+      "type": "object",
+      "properties": {
+        "registryCredentials": {
+          "description": "Private container registry credentials for containers used by the Sessions of the Session Pool.",
+          "$ref": "#/definitions/SessionRegistryCredentials"
+        },
+        "containers": {
+          "description": "List of container definitions for the Sessions of the Session Pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SessionContainer"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "ingress": {
+          "description": "SessionPool Ingress configuration.",
+          "$ref": "#/definitions/SessionIngress"
+        },
+        "sessionAuthConfiguration": {
+          "description": "Auth configuration of the Sessions in this Session Pool.",
+          "$ref": "#/definitions/SessionAuthConfiguration"
+        }
```

Comment:
Do we need to have an auth configuration here?

Reply:
Thanks, confirmed that this is removed, removing this now